implementations/rust/ockam/ockam_api/src/logs/logging_configuration.rs

```diff
@@ -249,7 +249,7 @@ pub fn logging_configuration(
     crates: CratesFilter,
 ) -> ockam_core::Result<LoggingConfiguration> {
     let is_legacy_variable_set = is_set::<LevelVar>(OCKAM_LOG)?;
-    let enabled = if is_legacy_variable_set {
+    let enabled = if is_legacy_variable_set || preferred_log_level.is_some() {
```

Comment:
The following won't print any logs, which is confusing, because the user is already showing the intent of enabling logging.

```rust
unset OCKAM_LOGGING
ockam status -vvv 
```

We should force the user to set both the `OCKAM_LOGGING` env var AND pass the verbose flag.